src/open_inwoner/ssd/templates/jaaropgave.html

```diff
@@ -2,136 +2,142 @@
-                    <th class="yearly-report__th">{{ jaaropgave.loon_zorgverzekeringswet.key }}</th>
-                    <th class="yearly-report__th">Ingehouden bijdrage Zorgverzekeringswet</th>
-                    <th class="yearly-report__th">{{ jaaropgave.periode.key }}</th>
-                </tr>
-                <tr class="yearly-report__tr">
-                    <td class="yearly-report__td">{{ client.bsn }}</td>
-                    <td class="yearly-report__td">{{ jaaropgave.loon_zorgverzekeringswet.value }}</td>
-                    <td class="yearly-report__td">0</td>
-                    <td class="yearly-report__td">{{ jaaropgave.periode.van }} t/m {{ jaaropgave.periode.tot }}</td>
-                </tr>
-            </table>
-        </div>
-        <!-- row 3 -->
-        <div class="subsection">
-            <table class="yearly-report__table table-fixed">
-                <colgroup>
-                    <col style="width: 23%;"/>
-                    <col style="width: 23%;"/>
-                    <col style="width: 23%;"/>
-                    <col style="width: 31%;"/>
-                </colgroup>
-                <tr class="yearly-report__tr">
-                    <!-- Hard-coded for now; see below -->
-                    <th class="yearly-report__th">Verrekende levensloopverlofkorting</th>
-                    <th class="yearly-report__th">Totaal premies werknemer-verzekeringen</th>
-                    <th class="yearly-report__th">{{ jaaropgave.werkgevers_heffing_premie.key }}</th>
-                    <th class="yearly-report__th">{{ jaaropgave.code_loonbelastingtabel.key }}</th>
-                </tr>
-                <tr class="yearly-report__tr">
-                    <td class="yearly-report__td">0</td>
-                    <td class="yearly-report__td">0</td>
-                    <td class="yearly-report__td">{{ jaaropgave.werkgevers_heffing_premie.value }}</td>
-                    <td class="yearly-report__td">{{ jaaropgave.code_loonbelastingtabel.value }}</td>
-                </tr>
-            </table>
-        </div>
-    </section>
-    <br/>
-    <!-- TOELICHTING -->
-    <section class="section section__toelichting extra-margin">
-        <div class="subsection extra-margin">
-            <p class="yearly-report__p yearly-report__comments" >{{ jaaropgave_comments|markdown|safe }}</p>
-        </div>
-    </section>
-</div>
-</body>
+{% for report in reports %}
+{% with client=report.client spec=report.specificatie inhoudingsplichtige=report.inhoudingsplichtige %}
+	<head>
+			<meta charset="UTF-8">
+			<meta name="viewport" content="width=device-width, initial-scale=1.0">
+			<title>Jaaropgave {{ spec.dienstjaar|slugify }}</title>
+	</head>
+	<body>
+	<div class="yearly-report">
+			<!-- HEADER -->
+			<!-- Title -->
+			<header class="header flex-container extra-margin-top">
+					<div class="flex-box heading">
+							<h1 class="h1">Jaaropgave {{ spec.dienstjaar|slugify }}</h1>
```

Comment:
This was meant as a temporary fix for incorrect output. For example, `dienstjaar` is printed as 2.022 instead of 2022 even though the data type of the value is `int`. It looks as if somewhere along the way from the data structure produced by xsdata and the template the value is converted to `float`, but I don't know where or why this happens.

Reply:
This didn't solve the problem.